ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/tools/ExposeAllLogsTest.java

```diff
@@ -106,24 +104,8 @@ private void test(boolean exposeAllLogsViaJMX) throws Exception
         assertFalse(uids2.containsKey(u));
     }
 
-    private void probeObjectStore(boolean exposeAllLogs, boolean useJMX) throws MBeanException {
-        ObjectStoreEnvironmentBean osEnvBean = BeanPopulator.getDefaultInstance(ObjectStoreEnvironmentBean.class);
-
-        osEnvBean.setExposeAllLogRecordsAsMBeans(exposeAllLogs);
-
-        ObjStoreBrowser osb = new ObjStoreBrowser();
-
-        // make sure the object store tooling MBean is ready
-        osb.start();
-
-        if (useJMX)
-            osb.setExposeAllRecordsAsMBeans(exposeAllLogs);
-
-        osb.probe();
-    }
-
     // Given a set of MBean names find their corresponding Uids
-    private Map<Uid, ObjectName> getUids(Map<Uid, ObjectName> uids, Set<ObjectName> osEntries) {
+    protected static Map<Uid, ObjectName> getUids(Map<Uid, ObjectName> uids, Set<ObjectName> osEntries) {
```

Comment:
Please can I ask you to clarify why the scope of the method needed to change? And was the same reason needed for the other getUids and the getProperty methods in this class?